test/TestProjects/MgmtMockAndSample/src/Generated/Models/MhsmNetworkRuleSet.Serialization.cs

```diff
@@ -58,7 +59,7 @@ internal static MhsmNetworkRuleSet DeserializeMhsmNetworkRuleSet(JsonElement ele
             }
             Optional<NetworkRuleBypassOption> bypass = default;
             Optional<NetworkRuleAction> defaultAction = default;
-            Optional<IList<MhsmipRule>> ipRules = default;
+            Optional<IList<DataFactorySecretString>> ipRules = default;
```

Comment:
This (and other similar instances) should not be changing. We will need to figure out a way to avoid these false positives that are caused by DataFactorySecretString having only a single Value property.